ci-operator/step-registry/ipi/conf/ovirt/generate-install-config/ipi-conf-ovirt-generate-install-config-ref.yaml

```diff
@@ -0,0 +1,12 @@
+ref:
+  as: ipi-conf-ovirt-generate-install-config
+  from: base
```

Comment:
For images that do not need anything special it's better to do:

```
  from_image:
   namespace: origin
   name: centos
   tag: '8'
```

This has an advantage of not needing the client ci-op config to provide a `base` image only to run the steps (see https://github.com/openshift/release/pull/14435 for more information - we want to do do this in the whole repo)